project_scrum/models/project_task.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+# Copyright <2017> <Tenovar Ltd>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models, fields, api
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+    _order = "sequence"
+
+    @api.model
+    def create(self, vals):
+        vals['reference'] = self.env['ir.sequence'].next_by_code(
```

Comment:
check if this is created within a project that has `use_scrum` activated

Reply:
or (better) remove entirely and depend on https://github.com/OCA/project/tree/10.0/project_task_code